Document-Processing/PDF/PDF-Library/NET/Merge-Documents.md

```diff
@@ -212,10 +221,10 @@ You can import multiple pages from an existing document by using [ImportPageRang
 {% tabs %}
 {% highlight c# tabtitle="C# [Cross-platform]" playgroundButtonLink="https://raw.githubusercontent.com/SyncfusionExamples/PDF-Examples/master/Merge%20PDFs/Import-multiple-pages-from-an-existing-PDF/.NET/Import-multiple-pages-from-an-existing-PDF/Program.cs" %} 
 
+using Syncfusion.Pdf;
```

Comment:
Here used PdfLoadedDocument, but the namespace is Syncfusion.Pdf:

Have you checked the github sample and added the required namespaces for it? if not verify and add all the necessary namespaces

Reply:
Updated proper namespace